src/react-components/footer.js

```diff
@@ -57,6 +62,22 @@ export default class Footer extends Component {
                   <strong>Invite</strong> Others
                 </span>
               </button>
+              <button className={styles.menuButton} onClick={this.props.onClickHelp}>
+                <i className={styles.menuButtonIcon}>
+                  <FontAwesomeIcon icon={faQuestion} />
+                </i>
+                <span className={styles.menuButtonText}>
+                  <strong>Learn</strong> How to Play
+                </span>
+              </button>
+              <button className={styles.menuButton} onClick={this.props.onClickUpdates}>
+                <i className={styles.menuButtonIcon}>
+                  <FontAwesomeIcon icon={faNewspaper} />
+                </i>
+                <span className={styles.menuButtonText}>
+                  <strong>Sign Up</strong> for Updates
+                </span>
+              </button>
```

Comment:
This makes the menu too tall in landscape on an S5. The close button gets hidden above the top of the screen.